src/components/Header/Header.scss

```diff
@@ -0,0 +1,39 @@
+$headerHeightDesktop: 160px;
+$headerHeightMobile: 64px;
+.header {
+  display: flex;
+  flex-direction: row;
+  background-color: #fe9436;
+  justify-content: center;
+  align-items: center;
+  margin: auto 0;
+  @media screen and (max-width: 480px) {
+    height: $headerHeightMobile;
+  }
+  @media screen and (min-width: 481px) {
+    height: $headerHeightDesktop;
+  }
+  .header-title {
+    font-size: calc(14px + 2vmin);
+    flex: 1;
+    color: white;
+    text-align: center;
+    @media screen and (max-width: 480px) {
+      margin-left: $headerHeightMobile;
+    }
+    @media screen and (min-width: 481px) {
+      margin-left: $headerHeightDesktop;
+    }
+  }
+  .header-logo {
+    img {
+      @media screen and (max-width: 480px) {
+        height: $headerHeightMobile;
+      }
+      @media screen and (min-width: 481px) {
+        height: $headerHeightDesktop;
+      }
+      margin-top: 4px; //to fit the image to box size (discover why it has more 4px)
```

Comment:
Por conta desse `margin-top: 4px` a image fica fora do header, veja imagem abaixo. Poderia confirmar como ficaria se vc excluir essa linha?

![Screenshot 2023-03-01 at 10 13 59 pm](https://user-images.githubusercontent.com/7925408/222137092-5c129539-ea06-4f21-91d7-e57b53345570.png)

Reply:
Você tem que clicar na imagem para ver o detalhe

Reply:
I don't know why, but in my desktop the image seems to be aligned with the header, i wrote this line because it was looking like there was a white space on the bottom of the image.
If you look the image below it seems to be correct, but i'll try to solve this problem
![image](https://user-images.githubusercontent.com/125293546/222169937-ac79b845-d74b-4e79-8f4d-ad67c9d5df33.png)

Reply:
Deleted the line and still seems to be ok, so i'll make the sugested changes.